bazelisk.py

```diff
@@ -306,6 +306,8 @@ def download_bazel_into_directory(version, is_commit, directory):
 
     sha256_path = destination_path + ".sha256"
     expected_hash = ""
+    matcher=re.compile(r"(\d*\.\d*(?:\.\d*)?)(rc\d+)?")
```

Comment:
Can you please format the changes? I think running black should work.

Reply:
Have run black to reformat. Can you please take a look? Thanks!

Reply:
@fweikert Hi, is there anything else I need to do for this PR? Appreciate a lot if could be merged.